tests/test_executor.py

```diff
@@ -66,6 +66,25 @@ def transformed_model_with_observed():
     return transformed_model_with_observed
 
 
+@pytest.fixture("module")
+def class_model():
+    class PyMC4ClassModel:
+        @pm.model
+        def class_model_method(self):
+            norm = yield pm.Normal("n", 0, 1)
+            return norm
+
+    return PyMC4ClassModel()
+
+
+def test_class_model(class_model):
```

Comment:
Is this what it's supposed to look like? It works but its pretty ugly imo. Am I missing the more "pythonic" version?